File: 1. Model/Telefone.py


class Telefone:

    def __init__(self, telefonePai, telefoneMae, telefoneOutros):
        self.telefonePai    = telefonePai
        self.telefoneMae    = telefoneMae
        self.telefoneOutros = telefoneOutros
        
        





    #  - - - - - - - - - - GETS & SETS - - - - - - - - - -
    ##GETS
    def getTelefonePai(self):
        return self.telefonePai

    def getTelefoneMae(self):
        return self.telefoneMae

    def getTelefoneOutros(self):
        return self.telefoneOutros


    ##SETS
    def setTelefonePai(self, tp):
        self.telefonePai = tp

    def setTelefoneMae(self, tm):
        self.telefoneMae = tm

    def setTelefoneOutros(self, to):
        self.telefoneOutros = to
    
    # - - - - - - - - - - //// /  //// - - - - - - - - - -

File: model/Telefone.java
package model;



public class Telefone {

    private int telefonePai;
    private int telefoneMae;
    private int telefoneOutros;



    public Telefone(int p, int m, int o) {
        this.telefonePai    = p;
        this.telefoneMae    = m;
        this.telefoneOutros = o;
    }



    // Gets
    public int getTelefonPai() {
        return this.telefonePai;
    }

    public int getTelefoneMae() {
        return this.telefoneMae;
    }

    public int getTelefoneOutros() {
        return this.telefoneOutros;
    }



    // Sets
    public void setTelefonePai(int p) {
        this.telefonePai = p;
    }

    public void setTelefoneMae(int m) {
        this.telefoneMae = m;
    }

    public void setTelefoneOutros(int o) {
        this.telefoneOutros = o;
    }
}
